api4/bot_local.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package api4
+
+func (api *API) InitBotLocal() {
+	api.BaseRoutes.Bot.Handle("", api.ApiLocal(patchBot)).Methods("PUT")
+	api.BaseRoutes.Bot.Handle("/disable", api.ApiLocal(disableBot)).Methods("POST")
+	api.BaseRoutes.Bot.Handle("/enable", api.ApiLocal(enableBot)).Methods("POST")
+	api.BaseRoutes.Bot.Handle("/assign/{user_id:[A-Za-z0-9]+}", api.ApiLocal(assignBot)).Methods("POST")
+
+	api.BaseRoutes.Bots.Handle("", api.ApiLocal(getBots)).Methods("GET")
```

Comment:
Should we also include `api.BaseRoutes.Bot.Handle("", api.ApiLocal(getBot)).Methods("GET")`?

Reply:
`getBot` gets a bot by it's userID. `getBots` gets by ownerId (with other search options). I couldn't find a Jira ticket for `getBot`, so I assumed it wasn't high priority for this epic.

Reply:
I see, we can add this in another PR.